x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/pipelines_json_configurations.tsx

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { useActions, useValues } from 'kea';
+
+import {
+  EuiButtonEmpty,
+  EuiCodeBlock,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiFormRow,
+  EuiLink,
+  EuiNotificationBadge,
+  EuiSelect,
+  EuiSpacer,
+} from '@elastic/eui';
+
+import { i18n } from '@kbn/i18n';
+
+import { DataPanel } from '../../../../shared/data_panel/data_panel';
+import { docLinks } from '../../../../shared/doc_links';
+import { HttpLogic } from '../../../../shared/http';
+import { isManagedPipeline } from '../../../../shared/pipelines/is_managed';
+
+import { PipelineJSONBadges } from './pipeline_json_badges';
+import { IndexPipelinesConfigurationsLogic } from './pipelines_json_configurations_logic';
+
+export const PipelinesJSONConfigurations: React.FC = () => {
+  const { http } = useValues(HttpLogic);
+  const { pipelineNames, selectedPipeline, selectedPipelineId, selectedPipelineJSON } = useValues(
+    IndexPipelinesConfigurationsLogic
+  );
+  const { selectPipeline } = useActions(IndexPipelinesConfigurationsLogic);
+  return (
+    <>
+      <EuiSpacer />
+      <DataPanel
+        hasBorder
+        title={
+          <h2>
+            {i18n.translate(
+              'xpack.enterpriseSearch.content.indices.pipelines.tabs.jsonConfigurations.title',
+              { defaultMessage: 'Pipeline configurations' }
+            )}
+          </h2>
+        }
+        subtitle={i18n.translate(
+          'xpack.enterpriseSearch.content.indices.pipelines.tabs.jsonConfigurations.subtitle',
+          { defaultMessage: 'View the JSON for your pipeline configurations on this index.' }
+        )}
+        footerDocLink={
+          <EuiLink href={docLinks.ingestPipelines} target="_blank" color="subdued">
```

Comment:
I'm not 100% sure if we want to use the same doc link here since it's already in the footer for the Ingest Pipelines card.